programs/marginfi/src/state/price.rs

```diff
@@ -31,16 +31,22 @@ pub enum PriceBias {
     High,
 }
 
+#[derive(Copy, Clone, Debug)]
+pub enum OraclePriceWeightType {
```

Comment:
let's call it OraclePriceType, since weight is already a charged word in this codebase